app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,11 @@
 
   <body>
     <%= render partial: 'layouts/header.html.erb' %>
+    <% if !flash.empty? %>
+      <% if !flash[:error].nil? %>
```

Comment:
try to always ask for the positives, it's clearer: `if flash[:error].present?`. Also you can merge the ifs